app/employee-app/backend/modules/types/constants.bal

```diff
@@ -0,0 +1,27 @@
+//
+// Copyright (c) 2005-2024, WSO2 LLC. (http://www.wso2.org) All Rights Reserved.
+//
+// WSO2 Inc. licenses this file to you under the Apache License,
+// Version 2.0 (the "License"); you may not use this file except
+// in compliance with the License.
+// You may obtain a copy of the License at
+// 
+// http://www.apache.org/licenses/LICENSE-2.0
+// 
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+// 
+
+public const APP_NAME = "Employee App";
+public const int DEFAULT_LIMIT = 1000;
```

Comment:
Shall we use a single convention here? Some constants have a type, some don't. It would be better to follow a single pattern.

Reply:
ACK